ChangeLog

```diff
@@ -17,6 +17,13 @@ Release date: TBA
 
   Closes #4301
 
+* Improved recognition of namespace packages. They should no longer be added to ``sys.path``
+  which should prevent various issues with false positives.
+  Because of the difficulities with mirroring the python import mechanics we would gladly
```

Comment:
```suggestion
  Because of the difficulties with mirroring the python import mechanics we would gladly
```

Reply:
Your review always make me wonder whether I paid any attention in school..

Reply:
We could activate our own spelling checker for that, let me try 😄